sentry.gradle

```diff
@@ -169,7 +202,21 @@ gradle.projectsEvaluated {
             group = 'sentry.io'
 
             delete sourcemapOutput
-            delete "$buildDir/intermediates/assets/release/index.android.bundle.map" // react native default bundle dir
+            delete "$buildDir/intermediates/assets/release/index.android.bundle.map"
+            // react native default bundle dir
+        }
+
+        def modulesCleanUpTask = tasks.create(name: nameModulesCleanup, type: Delete) {
+            description = "clean up collected modules generated file"
+            group = 'sentry.io'
+
+            delete modulesOutput
+        }
```

Comment:
M: Why not simply reuse the `cliCleanUpTask` task as suggested?

Reply:
Because it runs before the package task and the file will be deleted before it's included in the apk.